R/fit_process.R

```diff
@@ -51,6 +51,16 @@ spim_fit_process <- function(samples, parameters, data,
   message("Extracting demography")
   model_demography <- extract_demography(samples)
 
+  ## Check if severity calculations were outputted and, if so, extract
+  traj_names <- rownames(samples$trajectories$state)
+  if (all(c("ifr", "ihr", "hfr") %in% traj_names)) {
```

Comment:
better to do this with a switch I think - we also want to use the switch in lancelot_index

Reply:
done; I think I'm having the same trouble as Yasin to use lancelot_index switches, so I've set it to TRUE as default in sircovid